dist/lockup.js

```diff
@@ -215,6 +234,10 @@
 
         _preventDefault: function(e) {
             e.preventDefault();
+        },
+
+        _getStyle: function(e, property) {
+            return parseInt(e.css(property));
```

Comment:
The second radix parameter specifies the base (10/binary/octal, etc). Some browsers would parse "08" as octal. 

I don't know if this actually matters in practice, since [ES5 has removed the octal implementation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt). That same MDN article still recommends adding the radix, so... (shrug).